test/performance/alphabet/alphabet_to_rank_benchmark.cpp

```diff
@@ -49,4 +56,29 @@ BENCHMARK_TEMPLATE(to_rank_, seqan3::masked<seqan3::dna4>);
 BENCHMARK_TEMPLATE(to_rank_, seqan3::qualified<seqan3::dna4, seqan3::phred42>);
 BENCHMARK_TEMPLATE(to_rank_, seqan3::qualified<seqan3::dna5, seqan3::phred63>);
 
+#if SEQAN3_HAS_SEQAN2
+template <typename alphabet_t>
+void to_rank_2(benchmark::State & state)
+{
+    std::array<alphabet_t, 256> alphs{};
+    uint8_t i = 0;
+    for (alphabet_t & a : alphs)
+        a = static_cast<uint8_t>(i++ % seqan::ValueSize<alphabet_t>::VALUE);
```

Comment:
We should put that into its own function and let it use by both tests so we make sure, that we test the same in both cases.
```cpp
template <typename alphabet_t, bool is_seqan2>
std::array<alphabet_t, 256> create_alphabet_array(size_t const alphabet_size)
{
    std::array<alphabet_t, 256> alphabet_array{};

    auto convert_to_alphabet = [] (auto const rank) 
    {
        if constexpr (is_seqan2)
             return static_cast<uint8_t>(rank);
        else
            seqan3::from_rank(alphabet_t{}, rank);
    };

    for (size_t i = 0; i < alphabet_array.size(); ++i)
        alphabet_array[i] = convert_to_alphabet(i % alphabet_size);

    return alphabet_array;
}
```

And then call the function from both benchmarks.
This should be done similarly for the other three files above.